docs/installation.rst

```diff
@@ -40,16 +40,19 @@ If you are using Mamba, the steps are similar:
 Downloading and Installing STARDIS
 ----------------------------------
 
+.. note::
+   Both of the below instructions should be run with your ``stardis`` virtual environment activated. That is, you should run ``$ conda activate stardis`` or ``$ mamba activate stardis`` if you are using conda or mamba, respectively, in the terminal before you continue with the directions below. 
+
 For Non-Developers
 ^^^^^^^^^^^^^^^^^^
 
-STARDIS can be downloaded by **non-developers** by running
+STARDIS can be downloaded for non-developers by running
```

Comment:
I would still keep the non-developers as bold, so it's easier to notice

Reply:
[Done](https://github.com/tardis-sn/stardis/pull/109/commits/3a946dab7fba0de1e8ffe26594d9c7c553b988b9)